src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/loggers/PSFilterFileLogger.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq.loggers;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import org.apache.spark.api.java.JavaRDD;
+import org.broadinstitute.hellbender.metrics.MetricsUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Logs filtering read counts to metrics file
+ */
+public final class PSFilterFileLogger implements PSFilterLogger {
+
+    private final PSFilterMetrics metrics;
+    private final MetricsFile<PSFilterMetrics, Long> metricsFile;
+    private final String metricsOutputPath;
+
+    public PSFilterFileLogger(final MetricsFile<PSFilterMetrics, Long> metricsFile, final String metricsOutputPath) {
+        this.metrics = new PSFilterMetrics();
+        this.metricsFile = metricsFile;
+        this.metricsOutputPath = metricsOutputPath;
+    }
+
+    @Override
+    public void logPrimaryReads(final JavaRDD<GATKRead> reads) { metrics.PRIMARY_READS = reads.count(); }
+    public void logReadsAfterPrealignedHostFilter(final JavaRDD<GATKRead> reads) { metrics.READS_AFTER_PREALIGNED_HOST_FILTER = reads.count(); }
```

Comment:
Missing [at]Override.
Also please use 3 lines for each method.

Reply:
Done